### ISSUE TITLE
Define temporary folder or disable json response creating

### ISSUE THREAD

**Issue by skuda** [a85/POSTMan-Chrome-Extension#5493]
**Is your feature request related to a problem? Please describe.**
I am frustrated by the constant files with the filename format "${uuid}-response.json" Postman creates in my /tmp directory

**Describe the solution you'd like**
Being  able to disable this behavior

**Describe alternatives you've considered**
Being able to define a different directory to those JSON generated files, I use /tmp a lot to organize my temporary files and this is driving me crazy

**Comment by shaoran** [comment]
I agree. This week I was testing an API and used Postman a lot, and I also use `/tmp` to organize my files and I was shocked with the amount of files that postman created.
